### ISSUE TITLE
Render out observable

### ISSUE THREAD

**Issue by dy** [chrisguttandin/subscribable-things#68]
Just found out this awesome repo. Was collecting same cases in spectjs.
FYI - [spect/h](https://github.com/spectjs/spect#specth) can render observables to HTML directly, probably in a very if not the most efficient way. If that's of any help.

**Comment by chrisguttandin** [comment]
Wow great, I didn't see that library before. Do you think it makes to add an example to the [usage section](https://github.com/chrisguttandin/subscribable-things#usage) in the readme and [integration tests](https://github.com/chrisguttandin/subscribable-things/tree/master/test/integration) to make sure it works flawlessly?

**Comment by dy** [comment]
Looking forward. Something like
```
import h from 'spect/h.js'
import {animationFrame} from 'subscribable-things.js'

document.body.appendChild(h`<div>Frame #${animationFrame()}</div>`)
```
Do you have an ESM bundle? Seems that latest bundle has UMD format, no?

**Comment by dy** [comment]
Hmm, trying to import from unpkg.com, getting errors...
![image](https://user-images.githubusercontent.com/300067/139759027-621e6b38-2fcd-46c4-abc6-6a53469fb1eb.png)

**Comment by dy** [comment]
UPD. Codepen: https://codepen.io/dyv/pen/XWazNmy

**Comment by chrisguttandin** [comment]
There should be a UMD bundle at `/build/es5` but there is also an ESM bundle at `/build/es2019`.

**Comment by dy** [comment]
Right, but it doesn't seem to be a bundle ([module.js](https://npmfs.com/package/subscribable-things/2.0.0/build/es2019/module.js)), just a set of modules, right?

**Comment by chrisguttandin** [comment]
Yes, that's correct. Sorry for the confusion. It's transpiled but it's not a bundle (aka not a single file).

**Comment by chrisguttandin** [comment]
Thanks again for making me aware of spect. It's really a great library. I'll close this as it has been addressed by #69 and #70.

Please feel free to open another issue for the missing ESM bundle if that's important for you.
